docs/mta/README.md

```diff
@@ -8,6 +8,23 @@ At the end of a successful assessment workflow, a link to the report will be ava
 ## Configuration
 View the [MTA README on GitHub](https://github.com/parodos-dev/serverless-workflows-config/blob/main/charts/mta/README.md)
 
+## Prerequisites
+The token for sending notifications from the m2k workflow to RHDH notifications service needs to be provided to the workflow.
+
+To obtain the value for `NOTIFICATIONS_BEARER_TOKEN` use the value of the following command:
+```bash
+oc get secrets -n rhdh-operator backstage-backend-auth-secret -o go-template='{{ .data.BACKEND_SECRET | base64decode }}'
+```
+Or fetch the secret from the app-config.yaml of your RHDH instance if not installed by the Orchestrator operator.
+
+And create a secret named `${WORKFLOW_NAME}-creds` containing the token:
```

Comment:
the creation of the secret will be done as part of installing the chart.
the user should only put the required value in the secret.

Reply:
yes, I need to rework this PR after the changes made yesterday to https://github.com/parodos-dev/serverless-workflows/pull/315